### ISSUE TITLE
|include=#Section does not respect noinclude or onlyinclude tags

### ISSUE THREAD

**Issue by CrystalClearGal** [Universal-Omega/DynamicPageList3#162]
**DynamicPageList3 version**: 3.4.5
**MediaWiki version**: 1.37.2 (Miraheze)
**PHP version**: 7.4.28

TLDR: Problem observed here:
https://dpl3.miraheze.org/wiki/Criteria_for_page_selection/Parameter:_title_(example_1)

**List of steps to reproduce** (step by step, including full links if applicable):
* On an "Apple" page, make a Notes section with content, followed by a category for the page in either noinclude on includeonly tags.
* Put on another page:
```
{{#dpl:
|title=Apple
|include=#Notes
}}
```

**What happens?**:
The page with the dpl statement will be erroneously categorized because it ignores the noinclude/includeonly tags, as can be observed here:

https://dpl3.miraheze.org/wiki/Criteria_for_page_selection/Parameter:_title_(example_1)

**What should have happened instead?**:

DPL3 should be able to correctly noinclude/includeonly, and not cause unintended categorization.

**Browser information, screenshots and other applicable information**:

Not related.

**Comment by CrystalClearGal** [comment]
An update , I now see `|eliminate = categories` can be used to filter off categories, though I still suspect it is unintentional/undesired for DPL3 to ignore noinclude/onlyinclude tags. Though perhaps there's some reason for this behavior that I'm just not understanding (likely)? 

For the time being, I'll leave the example as-is for viewing, but I'll add an explanation where `include` is used for whole pages or sections, that `eliminate` may need to avoid unwanted categorization.

**Comment by CrystalClearGal** [comment]
I'm closing this, as the proper way (intended way back in the day) was to use the `eliminate` parameter. 

For example:
```
{{#dpl:
|title=Apple
|include=#Notes
|eliminate=categories
}}
```
As is shown working here:
https://dpl3.wikitide.org/wiki/Help:162

**Comment by CrystalClearGal** [comment]
Looks like this is working for a new version of the extension that's in progress (in development). Awesome! I made a corresponding example here (hoping it might aid the dev): 
https://dpl3.wikitide.org/wiki/Help:162

([Special:Version](https://dpl3.wikitide.org/wiki/Special:Version) shows [DPL4](https://www.mediawiki.org/wiki/Extension:DynamicPageList4) and sharing docs for now, which makes sense!) 😄
